Qa fix legacy UI

mishautkin opened the pull request:
### Description

- Fix paypal popup behavior
- Update default country to USA
- Move specs from Germany to USA as default country
- Fix assertions according to new changes
- Update test title
- Lint fix
- Remove unused methos (PayPalUi -> savePaymentMethod)
- Fixes for PCP v3.0.8

mishautkin committed:
General updates

- update playwright-utils version.
- update timeout in playwright config

mishautkin committed:
Fix paypal popup behavior

mishautkin committed:
Update default country to USA

mishautkin committed:
Move specs from Germany to USA as default country

mishautkin committed:
Fix assertions according to new changes

mishautkin committed:
Update test title

mishautkin committed:
Lint fix

mishautkin committed:
Remove unused methos (PayPalUi -> savePaymentMethod)

mishautkin committed:
Fixes for PCP v3.0.8

sedatbsr commented on tests/qa-legacy-ui/package.json:
```diff
@@ -17,12 +17,9 @@
   "dependencies": {
     "@automattic/vip": "^3.14.0",
-    "@inpsyde/playwright-utils": "^2.6.1",
+    "@inpsyde/playwright-utils": "^2.7.0",
```
The latest version is 2.7.2. Will you update?

mishautkin replied:
Sure, will commit the change.

mishautkin committed:
General updates

- Update playwright-utils version in package.json
- Uncomment woocommerce.setup

mishautkin committed:
Un-gitignore .zip files in `resources/files`

mishautkin committed:
Merge branch 'qa' of https://github.com/woocommerce/woocommerce-paypal-payments into qa-fix-legacy-ui
